code/modules/research/design_database.dm

```diff
@@ -14,6 +14,7 @@ var/global/list/default_initial_tech_levels
 	icon_state = "blackbox"
 	density = TRUE
 	anchored = TRUE
+	construct_state = /decl/machine_construction/default/panel_closed
```

Comment:
You probably want to also set 
```
	uncreated_component_parts = null
	stat_immune = 0
```
(or, possibly, not include the extra components on the circuit if you prefer). May also be worth checking the power draw vars on it.